### DISCUSSION TITLE
right click shortcut doesn't work at all

### DISCUSSION THREAD

**Discussion by JacopoAngelini** [cyrilletuzi/vscode-angular-schematics#902] [Q&A] [answered]
hi there, i'm working on an angular 2  project an i just installed angular schematics extension but whe i right click on a folder it doesn't show generate: module, generate: component... 

i tried to reinstall all the extension and vs code too but nothing. i have angular.json file in all the project i'm managing. 

how to solve this?

below my folders
<img width="278" alt="Screenshot 2022-05-19 12 31 28" src="https://user-images.githubusercontent.com/95528659/169273694-1128986a-52bd-45ff-a791-74c20bc37a3b.png">

and the errors
<img width="730" alt="Screenshot 2022-05-19 12 45 21" src="https://user-images.githubusercontent.com/95528659/169276171-490037be-398a-4b78-9345-99a70ea1bfa3.png">

<img width="482" alt="Screenshot 2022-05-19 12 41 19" src="https://user-images.githubusercontent.com/95528659/169276202-09623f02-e178-4b1b-a2aa-aa7cb6af9075.png">

translation: command '...' returned an error (...)

**Comment by cyrilletuzi** [comment, marked as answer]
Hi,

Your Explorer screenshot seems to show that your project is not opened at the right level. From the [extension documentation](https://github.com/cyrilletuzi/vscode-angular-schematics/blob/main/walkthroughs/open-folder.md#open-an-angular-folder):

> For Angular CLI and this extension to work, the opened folder must be the root Angular folder, which means the folder with the angular.json file.

And not from a parent directory like it seems to be the case.

**Comment by BearCarpenter** [comment]
I don't know if this is related but I have a similar problem with the right click menu but in my case installing the previous version fixed the problem
Extensions -> Angular Schematics -> expand Uninstal -> Install Another Version
![image](https://user-images.githubusercontent.com/38364002/202145527-fa8a7f27-05f9-46d3-a328-8932f3f60955.png)
in the last version I didn't have the menu and now I have it.

**Reply by ben12** [reply]
Same for me, I'm in remote SSH opened to the angular root.

**Reply by cyrilletuzi** [reply]
There was no code change from version 5.2.6 to 5.2.8, just documentation. The only thing I can see is an update of the `vsce` dependency, which manages the build and publish of VS Code extensions. As I have seen some issues in other repos about this step, I've done a new 5.2.9 release with the very last `vsce` version to be sure. Tested it successfully both on Mac and Windows.

Please let me know if it fixed your issues. Otherwise, please open an issue and follow the issue reporting process, as I cannot help you if you do not provide any information.

**Reply by ben12** [reply]
It is fixed for me with the version 5.2.9 :) Thanks
